# The struct input_event is not y2038 safe.
# Update the struct according to the kernel patch:
# https://lkml.org/lkml/2018/1/6/324

--- lirc-0.10.1/plugins/devinput.c
+++ lirc-0.10.1.new/plugins/devinput.c
@@ -459,8 +459,8 @@
 		return 0;
 	}
 
-	log_trace("time %ld.%06ld  type %d  code %d  value %d", event.time.tv_sec, event.time.tv_usec, event.type,
-		  event.code, event.value);
+	log_trace("time %ld.%06ld  type %d  code %d  value %d", event.input_event_sec, event.input_event_usec,
+		  event.type, event.code, event.value);
 
 	value = (unsigned)event.value;
 #ifdef EV_SW
